relay-config/src/config.rs

```diff
@@ -643,6 +692,121 @@ impl Config {
         Ok(config)
     }
 
+    /// Override configuration with values coming from other sources (e.g. env variables or
+    /// command line parameters)
+    pub fn apply_override(
+        &mut self,
+        overrides: OverridableConfig,
+    ) -> Result<&mut Self, ConfigError> {
+        let relay = &mut self.values.relay;
+
+        if let Some(upstream) = overrides.upstream {
+            relay.upstream = upstream
+                .parse()
+                .map_err(|err: UpstreamParseError| ConfigError::new_field_error("upstream", err))?;
+        }
+
+        if let Some(host) = overrides.host {
+            //relay.host = IpAddr::from_str(&host).map_err(|err| {
+            relay.host = host
+                .parse()
+                .map_err(|err: AddrParseError| ConfigError::new_field_error("host", err))?;
+        }
+
+        if let Some(port) = overrides.port {
+            relay.port = u16::from_str_radix(port.as_str(), 10)
+                .map_err(|err| ConfigError::new_field_error("port", err))?;
+        }
+
+        let processing = &mut self.values.processing;
+        if let Some(enabled) = overrides.processing {
+            match enabled.to_lowercase().as_str() {
+                "true" | "1" => processing.enabled = true,
+                _ => processing.enabled = false,
```

Comment:
I see your point that you made in [this comment](https://github.com/getsentry/relay/pull/546/files#r409474592). Would you prefer to only set to false if there is `"false" | "0"` given explicitly?

Reply:
Yes, I would prefer that, I set "true"| "1" and "false" | "0" | "" to be false , anything else to be an error